lib/cms/update_page_views_task.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../google_analytics/api'
+
+class UpdatePageViewsTask
+  def self.run
+    log('Starting UpdatePageViewsTask')
+    google_analytics_results = GoogleAnalytics::API.fetch_article_page_views
+
+    Comfy::Cms::Page.all_english_articles.each do |article|
+      unless article.update_page_views google_analytics_results
```

Comment:
I think I would prefer this in a begin rescue block and also raise the exception in the update_page_views method and then log the error here. 
Also can we add parentheses around the method arguments for consistency.